lib/channel.js

```diff
@@ -28,7 +28,7 @@ class Channel {
 
     this._send = createSender(sourceId, currentWindow.parent)
 
-    currentWindow.addEventListener('message', (event) => {
+    currentWindow.addEventListener('message', event => {
```

Comment:
It would be nice to do prettier somewhere else. Are we 100% sure nothing was changed in the `lib/` itself (other than formatting)?

Reply:
Rollbacked all changes in `lib/`